.gitignore

```diff
@@ -19,3 +19,6 @@ coverage/*
 
 # Ignore application configuration
 /config/application.yml
+
+# Ignore all stored files
+/public/srts/*
```

Comment:
For clarity's sake, consider renaming this folder `/public/srt`, non-pluralized.